proto/path/qos/evm.proto

```diff
@@ -6,6 +6,16 @@ option go_package = "github.com/buildwithgrove/path/observation/qos";
 import "path/qos/jsonrpc.proto";
```

Comment:
I'm a fan of the structure / direction this is taking.

Thinking out loud (Non-actionable): I'm curious if we end up keeping it as is or re-factoring in the future, and can actually see it going in either direciton.